lua/kommentary/config.lua

```diff
@@ -71,6 +71,25 @@ function M.has_filetype(filetype)
     return M.config[filetype] ~= nil
 end
 
+function M.config_from_commentstring(commentstring)
+    if commenstring == "/*%s*/" then return default end
+
+    local placeholder = '%s'
+    local where = commentstring:find(util.escape_pattern(placeholder))
+
```

Comment:
```suggestion
    if commenstring == "/*%s*/" then return default end
    local placeholder = '%s'
    local where = commentstring:find(util.escape_pattern(placeholder))
```
Remove unnecessary spaces to be consistent with the rest of the code.